rasa/core/test.py

```diff
@@ -15,30 +15,21 @@
 
 if typing.TYPE_CHECKING:
     from rasa.core.agent import Agent
+    from rasa.core.processor import MessageProcessor
 
-import matplotlib
 
+CONFUSION_MATRIX_STORIES_FILE = "story_confusion_matrix.png"
+REPORT_STORIES_FILE = "story_report.json"
 FAILED_STORIES_FILE = "failed_stories.md"
-
-# At first, matplotlib will be initialized with default OS-specific available backend
-# if that didn't happen, we'll try to set it up manually
-if matplotlib.get_backend() is not None:
-    pass
-else:  # pragma: no cover
-    try:
-        # If the `tkinter` package is available, we can use the `TkAgg` backend
-        import tkinter
-
-        matplotlib.use("TkAgg")
-    except ImportError:
-        matplotlib.use("agg")
+SUCCESSFUL_STORIES_FILE = "successful_stories.md"
 
 
 logger = logging.getLogger(__name__)
 
-StoryEvalution = namedtuple(
+StoryEvaluation = namedtuple(
     "StoryEvaluation",
-    "evaluation_store failed_stories action_list in_training_data_fraction",
+    "evaluation_store failed_stories successful_stories action_list "
+    "in_training_data_fraction",
```

Comment:
Let's change this to a list of strings, so that we can avoid having to manually split the string across lines leaving a space at the end. e.g `["evaluation_store", "failed_stories", ...]`